core/variable_operations.cpp

```diff
@@ -313,6 +313,18 @@ Variable y(const VariableConstView &pos) {
 Variable z(const VariableConstView &pos) {
   return transform<std::tuple<Eigen::Vector3d>>(pos, element::geometry::z);
 }
+Variable rotate(const VariableConstView &pos, const VariableConstView &rot) {
+  return transform<std::tuple<std::tuple<Eigen::Vector3d, Eigen::Quaterniond>>>(
```

Comment:
You specified the supported types twice: Once on the functor `element::geometry::rotate`, and once here. -> Remove template args from `transform` here.

Reply:
Depends on #1048 , so will be fixed once that is merged

Reply:
ok I removed them now